src/hvsampledata/__init__.py

```diff
@@ -99,6 +101,85 @@ def penguins(
+    engine_kwargs : dict[str, Any], optional
+        Additional kwargs to pass to `read_csv`, by default None.
+    lazy : bool, optional
+        Whether to load the dataset in a lazy container, by default False.
+
+    Description
+    -----------
+    This dataset from the USGS Earthquake Catalog provides detailed information on global seismic events,
+    including parameters such as date, time, location (latitude and longitude), depth, magnitude, and event type.
+    This comprehensive resource supports earthquake monitoring, research, and hazard assessment efforts.
+    Retrieved from https://earthquake.usgs.gov/earthquakes
+
+    Schema
+    ------
+    | name        | type       | description                                                         |
+    |:------------|:-----------|:--------------------------------------------------------------------|
+    | time        | datetime   | Time when the event occurred.                                       |
+    | latitude    | float      | Decimal degrees latitude. Negative values for southern latitudes.   |
+    | longitude   | float      | Decimal degrees longitude. Negative values for western longitudes   |
+    | depth       | float      | Depth of the event in kilometers.                                   |
+    | depth_class | object     | The depth category derived from the depth column.                   |
+    | mag         | float      | The magnitude for the event.                                        |
+    | mag_class   | object     | The magnitude category derived from the mag column.                 |
+    | place       | object     | Textual description of named geographic region near to the event.   |
+
+    Source
+    ------
+    `earthquake.csv` dataset courtesy of the U.S. Geological Survey
+    https://www.usgs.gov/programs/earthquake-hazards, with 6 months of data
+    selected from Jan. to Jun., 2024 along the so called Pacific Ring of Fire region.
+
+    License
+    -------
+    U.S. Public domain
+    Data available from U.S. Geological Survey, National Geospatial Program.
+    Visit the USGS at https://usgs.gov.
+
+    """
+    if engine == "polars":
+        engine_kwargs = {"null_values": "NA"} | (engine_kwargs or {})
+    tab = _load_tabular(
+        "earthquake.csv",
+        format="csv",
+        engine=engine,
+        engine_kwargs=engine_kwargs,
+        lazy=lazy,
+    )
+    # Convert the 'time' column to datetime
+    if engine == "pandas":
+        import pandas as pd
+
+        tab["time"] = pd.to_datetime(tab["time"], format="mixed", utc=True)
+    elif engine == "dask":
+        import dask.dataframe as dd
+
+        tab["time"] = dd.to_datetime(tab["time"], format="mixed", utc=True)
+    elif engine == "polars":
+        import polars as pl
+
+        tab = tab.with_columns(
+            pl.col("time").str.strptime(pl.Date, format="%Y-%m-%d %H:%M:%S%.f%z", strict=False)
+        )
+
+    return tab
```

Comment:
This uses built-in support for datetime conversion. The test likely also needs to be updated. 

```suggestion
    engine_kwargs = engine_kwargs or {}
    if engine == "polars":
        engine_kwargs = {"null_values": "NA", try_parse_dates=True} | engine_kwargs
    else:
        engine_kwargs = {"parse_dates": ["time"]} | engine_kwargs
    tab = _load_tabular(
        "earthquake.csv",
        format="csv",
        engine=engine,
        engine_kwargs=engine_kwargs,
        lazy=lazy,
    )
    return tab
```

Reply:
Also, I'm unsure if `"null_values": "NA"` is needed.

Reply:
Thanks